src/models/core/AdditionalHttpHeaderValues.ts

```diff
@@ -1,11 +1,20 @@
 
+/**
+ * AdditionalHttpHeaderValues allows users to specify additional values for specific HTTP headers.
+ *
+ * @public
+ **/
+export interface AdditionalHttpHeaderValues {
```

Comment:
do we anticipate there being more headers added in the future? otherwise I'm not sure how much value this interface adds

Reply:
Yes this is what Rose decided on, I raised the issue when she first had this idea, and then Tom also brought it up. I probably should have just written a comment saying we talked about this

Reply:
I see. So would this go all the way up to the Theme and Headless-React? Like we would want those libraries (and therefore the SDK and Headless) to also be able to specify other headers in the future instead of just passing in the Client-SDK header for now?

Reply:
Yes all the other libraries will have this updated config shape
The theme and headless-react will not expose this to the user though, at least not yet